Resources/Private/PlaceholderInsert/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "description": "PlaceholderInsert",
+    "license": "MIT",
+    "private": true,
+    "scripts": {
+        "build": "neos-react-scripts build",
+        "watch": "neos-react-scripts watch"
+    },
+    "devDependencies": {
+        "@neos-project/neos-ui-extensibility": "^1.3"
```

Comment:
really version 1.3?

Reply:
It doesn't matter really, it would compile with any version of the extensibility package. Will change to wildcard to avoid confusion.